### DISCUSSION TITLE
Key-point data annotation with CVAT

### DISCUSSION THREAD

**Discussion by anaghd** [ultralytics/ultralytics#5880] [Q&A]
I am new to Yolo and image processing. I have a question regarding image annotation for keypoint detection.
If I use a tool like CVAT to annotate keypoints, the tool allows me rotate bounding box. But eventually, since the Yolo format only cares about the center, the height and the width of the bounding box, does it even matter if the bounding box is rotated or not. Would it just be better to mark a bigger bounding box without rotation and encapsulate the whole object instead of rotating the bounding box and making the object fit exactly inside the bounding box?

**Comment by pderrenger** [comment]
@anaghd hello! For keypoint detection tasks, the orientation of the bounding box is indeed not relevant in the YOLO format. YOLO uses axis-aligned bounding boxes, which means that the boxes are not rotated and are defined by their center coordinates, width, and height.

When annotating keypoints, it's important to ensure that all the relevant keypoints are within the bounding box, regardless of its rotation. A larger, non-rotated bounding box that encapsulates the entire object is perfectly acceptable and often preferred for simplicity and compatibility with the YOLO format.

Remember that for keypoint detection, the precise locations of the keypoints are what's most important. The bounding box serves as a way to localize the object in the image, and the keypoints provide the detailed positions of specific parts of the object.

If you're working with YOLOv8 and keypoint detection, you can refer to the [Pose/Keypoint Estimation](https://docs.ultralytics.com/tasks/pose) section in our documentation for more guidance on how to prepare your data and train your model for this task. 😊🚀
